### PR TITLE
Benefits | Replace React radio button component with web components

### PR DESCRIPTION
**Pull request by Mottie**
## Summary

- *(Summarize the changes that have been made to the platform)*
  > Replacing React `RadioButtons` component with `va-radio` & `va-radio-option` in the following apps:
  > - HLR wizard (996)
  > - 526 wizard
  > - VRE 1900 wizard
  > - VRE 8832 wizard
  > Also fixed a few `aria-describedby` attributes which requires the dash & all lower case when used on web components
- *(If bug, how to reproduce)*
- *(What is the solution, why is this the solution)*
  > `RadioButtons` has been deprecated
- *(Which team do you work for, does your team own the maintainance of this component?)*
  > Benefits Decisions Reviews (formerly Benefits Team 1)
- *(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)*

## Related issue(s)

- [#52272](https://github.com/department-of-veterans-affairs/va.gov-team/issues/52272)
- [#1310](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1310)

## Testing done

- *Describe what the old behavior was prior to the change*
  > Updated appropriate unit & e2e tests
- *Describe the steps required to verify your changes are working as expected*
- *Describe the tests completed and the results*

## Screenshots

N/A - no visual change

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/static-pages/wizard/tests/bdd-526.unit.spec.jsx [hunk 1]
```diff
@@ -201,8 +202,9 @@ describe('Form 526 Wizard', () => {
     wrapper.unmount();
   });
 
-  // disagreeing flow
-  it('should allow disability form start when filing a new claim', () => {
+  // disagreeing flow - skipping since wizard is depreacted and 526 subtask will
+  // be implemented soon
+  it.skip('should allow disability form start when filing a new claim', () => {
```
Comment (Mottie) [inline src/applications/static-pages/wizard/tests/bdd-526.unit.spec.jsx:207]: Note that this entire wizard flow test is already covered by an e2e test - src/applications/disability-benefits/all-claims/tests/all-claims-wizard.cypress.spec.js

**Review by jtmst — APPROVED** [review]

**Review by adamwhitlock1 — APPROVED** [review]

**Review by rmessina1010 — APPROVED** [review]
LGTM

File: src/applications/vre/28-1900/wizard/pages/service-member/02-noVaMemorandum.jsx [hunk 2]
```diff
@@ -8,29 +8,38 @@ const options = [
...
-  />
-);
+const noVaMemorandum = ({ setPageState, state = {} }) => {
+  const handleValueChange = ({ detail } = {}) => {
+    const { value } = detail;
+    handleChangeAndPageSet(
+      setPageState,
+      value,
+      options,
+      'Are you in the Integrated Disability Evaluation System (IDES)or going through Physical Evaluation Board process?',
```
Comment (rocio-desantiago) [inline src/applications/vre/28-1900/wizard/pages/service-member/02-noVaMemorandum.jsx:18]: should there be a space after (IDES)?

File: src/applications/vre/28-8832/wizard/pages/DischargeStatus.jsx [hunk 3]
```diff
@@ -27,14 +29,25 @@ const DischargeStatus = ({ setPageState, state = {} }) => {
   };
 
   return (
-    <RadioButtons
-      name="discharge-status"
+    <VaRadio
+      id="dischargeStatus"
+      class="vads-u-margin-y--2"
       label="Have you or your sponsor been discharged form active-duty service in the last year, or do you have less than 6 months until discharge?"
```
Comment (rocio-desantiago) [inline src/applications/vre/28-8832/wizard/pages/DischargeStatus.jsx:35]: Found a typo unrelated to your changes. I suppose `form` should be `from`

**Review by rocio-desantiago — APPROVED** [review]
Two minor spelling comments. 
One could use a space.
The other is unrelated but noticed that form should be from.

File: src/applications/vre/28-1900/wizard/pages/service-member/02-noVaMemorandum.jsx (hunk 2 above)
Comment (Mottie) [reply src/applications/vre/28-1900/wizard/pages/service-member/02-noVaMemorandum.jsx]: Yep, thanks for catching this!

File: src/applications/vre/28-8832/wizard/pages/DischargeStatus.jsx (hunk 3 above)
Comment (Mottie) [reply src/applications/vre/28-8832/wizard/pages/DischargeStatus.jsx]: Another good catch, thanks!

**Pull request merged**